src/Admin/CitizenInitiativeAdmin.php

```diff
@@ -27,29 +24,29 @@ class CitizenInitiativeAdmin extends AbstractAdmin
         '_sort_by' => 'name',
     ];
 
-    public function setCitizenInitiativeManager(CitizenInitiativeManager $manager): CitizenInitiativeAdmin
+    public function setCitizenInitiativeManager(CitizenInitiativeManager $manager): self
     {
-        $this->manager = $manager;
+        $this->_manager = $manager;
 
         return $this;
     }
 
     public function getTemplate($name)
     {
-        if ('show' === $name) {
-            return 'admin/citizen_initiative/show.html.twig';
-        }
-
-        if ('edit' === $name) {
-            return 'admin/citizen_initiative/edit.html.twig';
-        }
-
-        return parent::getTemplate($name);
+        return $name === 'show' ?
+            'admin/citizen_initiative/show.html.twig' :
+            $name === 'edit' ?
+                'admin/citizen_initiative/edit.html.twig':
+                parent::getTemplate($name);
```

Comment:
Ternary operators should not be stacked.

http://php.net/manual/en/language.operators.comparison.php#example-104
> It is recommended that you avoid "stacking" ternary expressions. PHP's behaviour when using more than one ternary operator within a single statement is non-obvious: